tests/net/tcp/src/main.c

```diff
@@ -666,84 +659,99 @@ static bool test_register(void)
 		if (ret) {						\
 			DBG("TCP register %s failed (%d)\n",		\
 			    user_data.test, ret);			\
-			return false;					\
+			zassert_true(0, NULL);				\
 		}							\
 		user_data.handle = handlers[i++];			\
 		&user_data;						\
 	})
 
 #define REGISTER_FAIL(raddr, laddr, rport, lport)			\
+	do {								\
```

Comment:
Please fix indentation as you added do-while block, same comment to other macro changes below